### PR TITLE
source-slab contribution from aazam-gh

### LINKED ISSUE #46627: [connector-request] Slab
This is a **suggestion** to create a connector for **Slab** using the UI Connector Builder.
The website for the connector is https://slab.com.

If you want to try, claim this issue and start working on it following the steps below.

### Steps:
1. Comment in the issue and wait to be assigned to start working on it.
2. Map the API endpoints (get approval to move to 3 step)
3. Describe steps to get credentials.
4. Create the connector using the UI Builder

> [!IMPORTANT]
> Airbyte doesn't have sandbox credentials for this connector. You must have access/credentials to the service provider to create the connector.

> [!CAUTION]
> This is a **suggestion** and there may be cases where creating the connector using the Builder won't be possible. To minimize the risk of investing a lot of time directly in creating the connector, we strongly recommend to follow steps above.

### PR DESCRIPTION
**Pull request by aazam-gh**
## What

This PR adds a new connector for source Slab (source-slab).

Contributor-provided connector description:

Slab source connector for ingesting data from the slab API.
Slab is a platform to easily create, organise, and discover knowledge for your entire organisation, from non-technical to tech-savvy. https://slab.com/

In order to use this source, you must first create an account and log in. 
The API uses a bearer token which can be obtained by navigating to Settings -&gt; Developer -&gt; Admin Token.
You must be on a the business plan to obtain a token.
Slab uses Graphql API and this connector streams the `users`, `posts` and `topics` endpoints. Find more information about the API here https://studio.apollographql.com/public/Slab/variant/current/home 
## Details

<details><summary>Expand for list of spec properties and streams</summary>

### Configuration

| Input | Type | Description | Default Value |
|-------|------|-------------|---------------|
| `api_key` | `string` | API Key.  |  |
| `start_date` | `string` | start_date. Date from when the sync should start |  |

### Streams
| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
|-------------|-------------|------------|---------------------|----------------------|
| users | id | No pagination | ✅ |  ❌  |
| topics | id | No pagination | ✅ |  ❌  |
| posts | post_id | No pagination | ✅ |  ❌  |

</details>

## Reviewer checklist
- [ ] Run `/format-fix` to resolve any formatting errors
- [ ] Help the contributor upload an icon that follows our icon design guidelines
- [ ] Click into the CI workflows that wait for a maintainer to run them, which should trigger CI runs

### CODE REVIEW

File: airbyte-integrations/connectors/source-slab/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+        requester:
+          $ref: "#/definitions/base_requester"
+          path: graphql
+          http_method: POST
+          request_body_json:
+            query: >-
+              query { search(query: "posts" ,first: 100,  types: [POST]) { edges
+              { node { ... on PostSearchResult { post { id title content
+              insertedAt publishedAt archivedAt owner { id name email } topics {
+              id name } } } } }  } }
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-slab/manifest.yaml:103]: I don't believe it GraphQL connectors are supported by the builder today. With this limitation it wn't be possible to accept.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-slab/manifest.yaml (hunk 1 above)
Comment (aazam-gh) [reply airbyte-integrations/connectors/source-slab/manifest.yaml:103]: Ah ss that so :( the endpoints are being streamed

**Comment by marcosmarxm** [comment]
Hello 👋 Due to the numerous contributions happening, I decided to create this [view](https://github.com/orgs/airbytehq/projects/104/views/3) to help you know when I plan to review your contribution. If your connector has some limited-time credential feel free to reach me out.

**Comment by marcosmarxm** [comment]
The connector builder doesn't support properly GraphQL API endpoints for that reason this contribution was declinded.

**Pull request closed without merging**